.github/workflows/docker-nightly.yml

```diff
@@ -0,0 +1,33 @@
+name: nightly docker image builds
+
+on:
+  push:
+  schedule:
+    - cron: '0 6 * * *'
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: docker/setup-buildx-action@v3
+        with:
+          driver: docker
+      - uses: actions/checkout@v4
+        with:
+          repository: gazebosim/gz-sim
+          ref: gz-sim9
```

Comment:
should this be `main`?

Reply:
good catch: https://github.com/gazebosim/gz-sim/pull/2826/commits/075192575bef939f327ed85c9bd8dc4448323c9e